worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,191 @@
+[*Player options*](/games/Starcraft%202/player-options) ou en cliquant sur *Generate template* après avoir éxécuté le 
+*Launcher* d'Archipelago (i.e., `ArchipelagoLauncher.exe`). Ce modèle de base inclut une description pour chacune des 
+options et vous n'avez qu'à modifier les options dans un éditeur de texte de votre choix.
+* Vous pouvez demander à quelqu'un d'autre de partager un de ces fichiers *yaml* pour l'utiliser ou l'ajuster à vos 
+préférences.
+
+Prenez soin de vous rappeler du nom de joueur que vous avez inscrit dans la page à options ou dans le fichier *yaml* 
+puisque vous en aurez besoin pour vous connecter à votre monde!
+
+Notez que la page *Player options* ne permet pas de définir certaines des options avancées, e.g., l'exclusion de 
+certaines unités ou de leurs améliorations. 
+Utilisez la page [*Weighted Options*](/weighted-options) pour avoir accès à ces dernières.
+
+Si vous désirez des informations et/ou instructions générales sur l'utilisation d'un fichier *yaml* pour Archipelago, 
+veuilliez consutler [*Creating a YAML*](/tutorial/Archipelago/setup/en#creating-a-yaml).
+
+### Questions récurrentes à propos du fichier yaml
+#### Comment est-ce que je sais que mon *yaml* est bien défini?
+
+La manière la plus simple de valider est d'essayer. 
+Sauvegarder votre *yaml* dans le dossier `Players/` de votre installation d'Archipelago et exécuter 
+`ArchipelagoGenerate.exe`. 
+Si votre *yaml* est bien défini, vous devriez voir un nouveau fichier, avec l'extension `.zip`, apparaître dans le 
+dossier `output/` de votre installation d'Archipelago.
+Il est recommandé de lancer `ArchipelagoGenerate.exe` via un terminal afin que vous puissiez voir les messages générés 
+par le logiciel, ce qui va inclure toutes erreurs qui ont eu lieu et le nom de fichier généré.
+Si vous n'appréciez pas le fait d'utiliser un terminal, vous pouvez aussi regarder le fichier *log* qui va être produit 
+dans le dossier `logs/`.
+
+#### À quoi sert l'option *Progression Balancing*?
+
+Pour un monde *Starcraft 2* seule, cette option ne fait rien. 
+Il s'agit d'une option d'Archipelago qui permet de balancer la progression d'un monde relativement aux autres mondes en 
+interchangeant les *items* de progression dans les *sphères*.
+Si le *Progression Balancing* d'un monde est plus grand que ceux des autres, les *items* de progression de ce monde ont 
+plus de chance d'être obtenus tôt et vice-versa si sa valeur est plus petite que celle des autres mondes.
+Cependant, *Starcraft 2* est beaucoup plus permissif en termes d'*items* qui sont nécessaires pour progresser.
+Pour cette raison, cet ajustement a souvent peu d'influence sur la capacité de progresser dans un monde de 
+*StarCraft 2*. 
+Notez que l'utilisation de cette option, i.e. au moins un monde avec une valeur différente de *Progression Balancing*, 
+augmente le temps de génération d'un *MultiWorld*, alors certains recommandent de ne pas toucher à cette option.
```

Comment:
certains recommendent de la laisser à zéro*

Reply:
The default is 50. So, you suggest that users of SC2 use 0 which, if memory serves right, simply deactivate balancing for that word? 
If yes, I propose "certains recommendent de le définir à zéro."

Reply:
Well, using the default of 50 doesn't fit the rest of the sentence. 50 Will increase the generation time, as will any non-zero value, as it adds a step during generation.

So either the remainder of the sentence should be adjusted to properly explain that leaving it default is "fine", without claiming it is faster, or the recommendation should be zero, as zero is the fast value. Either is fine in my opinion, as the difference will not be noticeable to average players.

Reply:
Modified so that recommendation is zero.